install/kubernetes/cilium/values.yaml

```diff
@@ -434,4 +436,9 @@ global:
     #
     # See https://github.com/cilium/hubble/blob/master/Documentation/metrics.md for more comprehensive
     # documentation about Hubble's metric collection.
-    metrics: []
+    metrics:
+      enabled:
+       - drop
+      serviceMonitor:
```

Comment:
I wonder if we should be using the 'prometheus.serviceMonitor' instead of having one under the 'hubble/metrics'.
Please let me know your thoughts?

Reply:
Never mind, since we need to have the hubble ui separate, it makes sense to have it under the hubble section.

Reply:
Yeah, it's actually not related to hubble-ui at all!